tests/test_config.py

```diff
@@ -1518,6 +1518,14 @@ def test_verbosity(self, newconfig):
         config = newconfig(["-vv"], "")
         assert config.option.verbosity == 2
 
+    def test_quiet(self, newconfig):
```

Comment:
this should be  a parametrized test

Reply:
Ah, I see what you mean. I was basing it off the one above. Great catch.